00_hello_world/main.go

```diff
@@ -9,5 +9,20 @@ import (
 var out io.Writer = os.Stdout
 
 func main() {
-	fmt.Fprintln(out, "Hallo du schöne Welt!")
+
+	fibonacciSeries(7)
+}
+
+func fibonacciSeries(number int) {
+	var firstNumber = 1
+	var secondNumber = 1
+	fmt.Fprintln(out, firstNumber)
+	fmt.Fprintln(out, secondNumber)
+	for count := 3; count <= number; count++ {
+		var sum = firstNumber + secondNumber
+		fmt.Fprintln(out, sum)
+		firstNumber = secondNumber
+		secondNumber = sum
```

Comment:
You can fold these into a single line: `firstNumber, secondNumber = secondNumber, sum`.

Reply:
Done. Thanks